sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClient.java

```diff
@@ -819,26 +761,24 @@ public TextAnalyticsServiceVersion getServiceVersion() {
      * <p>Extract key phrases in a text with a provided language. Subscribes to the call asynchronously and prints
      * out the key phrases when a response is received.</p>
      *
-     * {@codesnippet com.azure.ai.textanalytics.TextAnalyticsAsyncClient.extractKeyPhrasesWithResponse#string-string}
+     * {@codesnippet com.azure.ai.textanalytics.TextAnalyticsAsyncClient.extractKeyPhrases#string-string}
      *
      * @param text the text to be analyzed.
      * @param language The 2 letter ISO 639-1 representation of language for the text. If not set, uses "en" for
      * English as default.
      *
-     * @return A {@link Mono} containing a {@link Response} whose {@link Response#getValue() value} has the
-     * {@link ExtractKeyPhraseResult key phrases} of the text.
+     * @return A {@link PagedFlux} containing the key phrases of the text.
      *
      * @throws NullPointerException if {@code text} is {@code null}.
-     * @throws com.azure.ai.textanalytics.models.TextAnalyticsException if the response returned with
-     * an {@link com.azure.ai.textanalytics.models.TextAnalyticsError error}.
+     * @throws TextAnalyticsException if the response returned with an {@link TextAnalyticsError error}.
      */
-    @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<Response<ExtractKeyPhraseResult>> extractKeyPhrasesWithResponse(String text, String language) {
+    @ServiceMethod(returns = ReturnType.COLLECTION)
+    public PagedFlux<String> extractKeyPhrases(String text, String language) {
```

Comment:
Why do we need to update these to `PagedResponses`?

Reply:
"DO return PagedFlux<T> for asynchronous APIs that expose collections. Even if the service does not support pagination, always return PagedFlux<T>, as it allows for consumers to retrieve response information in a consistent manner."

Reply:
I need to correct sync api to return IterableStream<T>

Reply:
update: Sync collection APIs will stay with `PageIterable`. It contains response infor